src/controllers/api/notes.js

```diff
@@ -0,0 +1,88 @@
+const { v4: uuidv4 } = require("uuid");
+const { getNotesFromFile, writeNotesToFile } = require("../../util.js");
+
+// get all notes
+const getNotes = (req, res) => {
+  const notes = getNotesFromFile();
+  if (notes.length) {
+    return res.json(notes);
+  }
+  return res.status(404).json({
+    message: `No notes found`,
+  });
+};
+
+// get specific note
+const getNoteById = (req, res) => {
+  const { id } = req.params;
+  const notes = getNotesFromFile();
+  const note = notes.find((note) => {
+    return note.id === id;
```

Comment:
you can also refactor this line since is only one line by removing the return keyword and curly bracket